src/test/resources/tough.golden.vcf

```diff
@@ -16,7 +16,92 @@
 ##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias">
 ##INFO=<ID=VQSLOD,Number=1,Type=Float,Description="Log odds ratio of being a true variant versus being false under the trained gaussian mixture model">
 ##INFO=<ID=culprit,Number=1,Type=String,Description="The annotation which was the worst performing in the Gaussian mixture model, likely the reason why the variant was filtered out">
-#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	tumor.chr20
+##contig=<ID=1,length=249250621,assembly=GRCh37>
```

Comment:
yea, it's what is generated now. 

if there was a more narrow a way to test this i'd be open to it, but this seems like the most straightforward way to have an end-to-end test.